tfhe/benches/boolean/bench.rs

```diff
@@ -8,15 +8,16 @@ use tfhe::boolean::parameters::{
     BooleanParameters, DEFAULT_PARAMETERS, PARAMETERS_ERROR_PROB_2_POW_MINUS_165,
     PARAMETERS_ERROR_PROB_2_POW_MINUS_165_KS_PBS,
 };
-use tfhe::boolean::prelude::{BinaryBooleanGates, DEFAULT_PARAMETERS_KS_PBS};
+use tfhe::boolean::prelude::{BinaryBooleanGates, DEFAULT_PARAMETERS_KS_PBS, TFHE_LIB_PARAMETERS};
```

Comment:
Why don't we use `tfhe::boolean::parameters` import like above for the two parameters here ?